cfgov/v1/models/handlers.py

```diff
@@ -0,0 +1,78 @@
+from collections import OrderedDict
+from itertools import chain
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore.blocks.stream_block import StreamValue
+
+from ..atomic_elements import ATOMIC_JS
+
+
+class Handler(object):
+    def __init__(self, page, request):
+        self.page = page
+        self.request = request
+
+    def process(self, context):
+        raise NotImplementedError
+
+    # Retrieves the stream values on a page from it's Streamfield
+    def get_streamfield_blocks(self):
+        lst = [value for key, value in vars(self.page).iteritems()
+               if type(value) is StreamValue]
+        return list(chain(*lst))
```

Comment:
i'm not sure about this line.  there's a typecast to a list b/c itertools.chains returns an iterable.  chain is normally used to combine iterables.  i only see a single iterable here - lst.